### DISCUSSION TITLE
Filesystem - Not writeable (777 permissions)

### DISCUSSION THREAD

**Discussion by coffebar** [rhubarbgroup/.github#47] [Support (ROC)] [answered]
I did `sudo chmod -R 777 wp-content` 

> drwxrwxrwx 13 www-data dev       4096 Nov  5 15:40 wp-content/

and plugin still says 
`Filesystem - Not writeable`

**Comment by tillkruss** [comment]
I assume the owner is wrong.

**Comment by yatsukhnenko** [comment]
It might be selinux

**Comment by coffebar** [comment]
> It might be selinux

not in my case

**Comment by coffebar** [comment]
It may be related to call of [request_filesystem_credentials](https://github.com/rhubarbgroup/redis-cache/blob/3b64cb347acb18ecbd23a398d5787ee52ea5d86c/includes/class-plugin.php#L1341C24-L1341C63) that [Displays a form](https://developer.wordpress.org/reference/functions/request_filesystem_credentials/) to the user to request for their FTP/SSH details in order to connect to the filesystem. For my WP setup, this form is not filled by password without user. So this call may fail

**Comment by tillkruss** [comment]
Correct, for Redis Object Cache to toggle the drop-in it needs filesystem permissions. If you didn't configure them, then you'll see `Not writeable`.

In the next version you can use [`WP_REDIS_DISABLE_DROPIN_CHECK`](https://github.com/rhubarbgroup/redis-cache/commit/d658193837bee8a5668940aed3f1fd11f0b038af).

**Comment by emf-developer** [comment, marked as answer]
if you have ftp error like when trying to install a plugin, add this in wp-config.php:

`define('FS_METHOD', 'direct');
`
